.buildkite/run.sh

```diff
@@ -68,15 +68,12 @@ echo "--- Set custom git config"
 cp .buildkite/.gitconfig "$HOME/"
 cat "$HOME/.gitconfig"
 
-echo "--- Run cargo fmt"
-(cd proxy && time cargo fmt --all -- --check)
-
-echo "--- Run proxy lints"
-(cd proxy && time cargo clippy --all --all-features --all-targets)
-
 echo "--- Run proxy tests"
 (cd proxy && time cargo test --all --all-features --all-targets)
 
+echo "--- Run proxy docs"
+(cd proxy && time cargo doc --no-deps)
```

Comment:
Why don't we bundle this one with fmt and clippy?

Reply:
Ya, can do! :)